### DISCUSSION TITLE
Commafeed sync

### DISCUSSION THREAD

**Discussion by AtiusAmy** [spacecowboy/Feeder#170] [Ideas]
It would be pretty cool to be able to do that :D

**Comment by spacecowboy** [comment]
Do what? What's commafeed?

**Reply by AtiusAmy** [reply]
To uh sync to commafeed and have it updated (idk if this is enough explaination, I'm really bad at this sort of thing)

Commafeed is an RSS software https://www.commafeed.com/#/welcome

**Comment by spacecowboy** [comment]
In that case no. I have no plans to implement support for external services at this date, except Feeder's own sync service.

I don't read RSS feeds on my computer so I have no need for that.
